config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -410,6 +410,14 @@ periodics:
       # even on very powerful machines
       - name: BENCHTIME
         value: 1ns
+      # We need to constraint compute resources so all the tests
+      # finish approximately at the same time. More compute power
+      # can increase scheduling throughput and make consequent results
+      # incomparable.
+      resources:
+        requests:
+          memory: "1Gi"
+          cpu: 4
```

Comment:
Lets not set the memory limit so that it doesn't get killed if the limit is exceeded. I don't think we set memory limits in the default deployment.

Reply:
also, we need to set the limit for the cpu so that it gets throttled.

Reply:
It's resource requests, not limits. Though, I can remove it and have it set automatically.

Reply:
Done

Reply:
Right, we should set the limit for the cpu as I argued in the second comment.